mixer/adapter/svcctrl/expiringLRUCache_test.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package svcctrl
+
+import (
+	"testing"
+	"time"
+
+	"github.com/hashicorp/golang-lru/simplelru"
+	"github.com/jonboulle/clockwork"
+)
+
+func TestAddAndGet(t *testing.T) {
+	fakeClock := clockwork.NewFakeClock()
+	cache := newTestCache(2, time.Second, fakeClock, t)
+	// cache contents: {(1, 1)}
+	if evicted := cache.add(1, 1); evicted {
+		t.Errorf(`expiringLRUCache.add(1, 1) != false`)
+	}
```

Comment:
This error statement is really difficult to understand. Maybe ' should be false'? Similar with the other error msg.

Reply:
OK, change to a more descriptive language.